sdk/include/reactos/mc/bugcodes.mc

```diff
@@ -1557,7 +1557,7 @@ Severity=Success
 Facility=System
 SymbolicName=MANUALLY_INITIATED_CRASH
 Language=English
-MANUALLY_INITIATED_CRASH
+The end-user manually generated the crashdump.
```

Comment:
"end-user" --> "user" , and "crashdump" --> "crash dump".

Reply:
That's how it's typed in Windows itself. Although your suggestion is more proper.